### ISSUE TITLE
Bug - "Blank login page"

### ISSUE THREAD

**Issue by Playmountain** [scania/corporate-ui#568]
**Describe the bug**  
Users are greeted by a blank page when attempting to login using IE 11
<img width="960" alt="ie11" src="https://user-images.githubusercontent.com/5740214/84763572-b4d0e680-afcc-11ea-936f-3b150970b577.png">

**To Reproduce**   
_Steps to reproduce the behavior:_  
1. Open IE 11
2. Open developer console
3. Visit https://fg.ciam.devtest.aws.scania.com/auth/realms/scania/protocol/openid-connect/auth?client_id=mupp_foo_client&redirect_uri=https://s3-eu-west-1.amazonaws.com/ciam-demo-app/index.html&response_type=token id_token&scope=email openid profile&nonce=foobarQ
4. See blank page, note several 404 entries under the network tab

**Expected behavior**  
Users should be greeted by a page allowing them to select which supported idp they wish to authenticate with
<img width="960" alt="edge" src="https://user-images.githubusercontent.com/5740214/84763744-f792be80-afcc-11ea-96a8-8eb7a590810d.png">

**Version of Corporate-ui**  
 _3.5.6-1_

**Framework and version**  
_If you use any specific framework please provide information_
- Framework: Whatever Keycloak uses
- Version: 9.0.1

**Desktop information:**  
_Please complete the following information_
 - Browser IE 11
 - Version 11.1488.17134.0
 - OS: Windows 10

**Additional context**   
_Add any other context about the problem here._
IE 11 tries to fetch various components using corrupted urls:
ex webcomponents
https://fg.ciam.devtest.aws.scania.com/auth/realms/scania/protocol/openid-connect/undefined/auth/resources/3naox/login/scania-corporate-ui/node_modules/webcomponents.js/webcomponents-lite.js
Url should most likely be:
https://fg.ciam.devtest.aws.scania.com/auth/resources/3naox/login/scania-corporate-ui/node_modules/webcomponents.js/webcomponents-lite.js
Hard to tell if this is the problem or just a symptom of something else.

We are currently using the most recent, non-beta version of corporate-ui ,which provides the login component so we unable to upgrade to a later version.

**Contact information**
CIAM team group mailbox: iam.maintenance@scania.com
anders.x.ohlsson@scania.com
tobias.rang@scania.com
daniel.lekberg@scania.com

**Comment by helloimela** [comment]
The problem is possibly caused by this line in CUI https://github.com/scania/corporate-ui/blob/v3/Update_login_component/src/global/ts/core.ts#L140
```js
  window['static_root'] = (wv.props ? JSON.parse(wv.props)['--root'] : undefined) || helpers.urlInfo(scriptUrl).origin;
  window['version_root'] = (window['static_root'] + helpers.urlInfo(scriptUrl).pathname).replace('/js/corporate-ui.js', '');
```

It only happens in IE though, maybe it cannot find `wv.props`
Other solution could be, from your application, force-assign correct url in windows variable 
`window['static_root'] = https://fg.ciam.devtest.aws.scania.com `


IE
![image](https://user-images.githubusercontent.com/1199101/85558959-8b960300-b629-11ea-9e33-791f2dd02225.png)

Chrome
![image](https://user-images.githubusercontent.com/1199101/85559004-9781c500-b629-11ea-8663-78fc8ea1fbc6.png)

**Comment by helloimela** [comment]
Fixed in #571 released in 3.5.6-2

Note: Please consider upgrade your project to use corporate-ui 4. We will drop support corporate-ui 3 in the future. The main reason is corporate-ui 3 is using Polymer v1 which is not a web standard and not widely supported by major browsers. Chrome drop support since April 2019. Right now it works using polyfill (webcomponents.js). But we never know when Chrome will drop support for the polyfill.
For sustainable solution we strongly recommend to upgrade to version 4. . We can use template and not a login component, which is more flexible to maintain for your team. In addition, all UI updates will be released in version 4.
